hack/release_catalog_map.sh

```diff
@@ -6,10 +6,6 @@ release_catalog_map["release-4.19"]="v10.19"
 release_catalog_map["release-4.18"]="v10.18"
 release_catalog_map["release-4.17"]="v10.17"
 release_catalog_map["release-4.16"]="v10.16"
-release_catalog_map["release-4.15"]="v10.15"
```

Comment:
@wgahnagl  consider removing also the 4.15 tekton files

https://github.com/openshift/windows-machine-config-operator-fbc/blob/main/.tekton/windows-machine-config-operator-fbc-release-4-15-pull-request.yaml